plugins/freshdesk/plugin.spec.yaml

```diff
@@ -9,6 +9,7 @@ supported_versions: ["FreshDesk API v2"]
 vendor: rapid7
 support: rapid7
 status: []
+cloud_ready: true
```

Comment:
You should also include keyword cloud_enabled. Kindly refer to : https://github.com/rapid7/insightconnect-plugins/blob/cef055d91a6ee6d605c98794644063f40092bc53/plugins/azure_ad_admin/plugin.spec.yaml#L24

Reply:
@blaxminarayan-r7 Done